src/sentry/static/sentry/app/components/charts/percentageBarChart.jsx

```diff
@@ -0,0 +1,131 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import BarSeries from './series/barSeries.jsx';
+import BaseChart from './baseChart';
+import Tooltip from './components/tooltip';
+import XAxis from './components/xAxis';
+import YAxis from './components/yAxis';
+
+const FILLER_NAME = '__filler';
+
+/**
+ * A stacked 100% column chart over time
+ *
+ * See https://exceljet.net/chart-type/100-stacked-bar-chart
+ */
+export default class PercentageBarChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+
+    series: PropTypes.arrayOf(
+      PropTypes.shape({
+        seriesName: PropTypes.string,
+        data: PropTypes.arrayOf(
+          PropTypes.shape({
+            category: PropTypes.string,
+            value: PropTypes.number,
+          })
+        ),
+      })
+    ),
+
+    getCategoryName: PropTypes.func,
+    getValue: PropTypes.func,
+  };
+
+  static get defaultProps() {
+    return {
+      getCategoryName: ({category, value}) => category,
+      getValue: ({category, value}, total) =>
```

Comment:
Is there a reason for this to be overridden by a user? It seems that if someone passes in a different function, this wouldn't be a percentage bar chart anymore